### PR TITLE
Tokens

### PR DESCRIPTION
**Pull request by chrisarevir**
### Summary

There was a need for UX approved tokens, so I went and made them.
Following mocks from [here](https://procore.invisionapp.com/d/main/#/console/12113821/260644185/preview)

#### After

screen shot 2017-12-01 at 9 16 41 am

### CODE REVIEW

File: token/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Token
+
+# Change Log
+
+### 1.0.0
```
Comment (ben-burlingham) [inline token/README.md]: 0.0.0?

File: token/src/Token.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  width: 16px;
+
+  &:hover {
+    background-color: white;
+    color: #2574c6;
+    cursor: pointer;
+  }
+
+  &::after {
+    content: '\D7';
```
Comment (ben-burlingham) [inline token/src/Token.scss]: oops

File: token/README.md (hunk 1 above)
Comment (dancingshell) [reply token/README.md]: 0.1.0?

Comment (nickwmurphy) [reply token/README.md]: 100.0.0?

File: token/sandbox/sandbox.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+import Tokens from '../src';
+import sandboxStyles from './sandbox.scss';
+
+const text = 'Text';
+const otherText = 'Other Text';
+const onClear = id => console.log(`removing id ${id}`); // eslint-disable-line
+
+const tokens = (
+  <Fragment>
```
Comment (dancingshell) [inline token/sandbox/sandbox.jsx]: Wow fragments! 🎉

File: token/README.md (hunk 1 above)
Comment (ben-burlingham) [reply token/README.md]: 01101000 01100001 01101000 01100001

Comment (dancingshell) [reply token/README.md]: 1751214177?
screen shot 2017-12-01 at 3 34 02 pm

Comment (ben-burlingham) [reply token/README.md]: 65 108 109 111 115 116

**Pull request merged**